test/clearingHouse/ClearingHouse.accounting.xyk.test.ts

```diff
@@ -571,19 +568,21 @@ describe("ClearingHouse accounting verification in xyk pool", () => {
         it("funding-induced liquidation", async () => {
             // maker add liquidity
             await addOrder(fixture, maker, 100, 10000, lowerTick, upperTick)
-
             // taker open
             await q2bExactInput(fixture, taker, 150)
 
+            await accountBalance.mockMarkPrice(baseToken.address, parseEther("4"))
             // set index price to let taker pay funding fee
             mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
```

Comment:
I think this line can be deleted, right?

Reply:
Actually we can't. Mocking mark price here is to make account value to become under margin, and mocking index price here is to fix the value of founding payment.